cfgov/unprocessed/css/on-demand/job_listing_list.less

```diff
@@ -0,0 +1,16 @@
+@import (reference) '../main.less';
+
+.m-jobs-list .m-list__links {
+  .respond-to-min( @bp-sm-min, {
+    .m-list_item + .m-list_item {
+      margin-top: unit( ( @grid_gutter-width / 2 ) / @base-font-size-px, em );
+    }
+  } );
+
+  .m-list_link-subtext {
+    .u-link__colors( @black );
+    display: block;
+    margin-top: unit(2px / @base-font-size-px, em);
```

Comment:
Strange that this lost its spacing on copy/paste, but the margin-top unit on line 6 didn't.

Reply:
Yeah, my linting/autofixing setup has been in a non-ideal state since working on https://github.com/cfpb/consumerfinance.gov/pull/7363 and I have just never bothered fixing it because I figured that would be merged eventually

Reply:
I think 7363 just needs BEWD sign-off/buy in.. also, maybe a docs troubleshooting note about the GitHub Desktop issue.

Reply:
Also, does `yarn lint` autofix anything for you?

Reply:
Yeah, fixed the linting in f3ef8a1fe49beb7898b79c851aa14a230284a450